content/docs/2.4/operate/cluster.md

```diff
@@ -58,3 +58,14 @@ $env:KEDA_HTTP_DEFAULT_TIMEOUT=1000
 ```
 
 All applicable scalers will use this timeout. Setting a per-scaler timeout is currently unsupported.
+
+## Kubernetes Client Parameters
+
+The Kubernetes client config used within Keda Adapter can be adjusted by passing the following command line flags to the binary:
```

Comment:
```suggestion
The Kubernetes client config used within KEDA Adapter can be adjusted by passing the following command-line flags to the binary:
```

Reply:
`KEDA Adapter` -> `KEDA Metrics Adapter`

Reply:
Updated as per the comments